### PR TITLE
Alternative proposal for udata#3862

### PR DESCRIPTION
**Pull request by streino**
Alternative proposal, see https://github.com/opendatateam/udata/pull/3862#discussion_r3682859847

### CODE REVIEW

File: udata/rdf.py [hunk 1]
```diff
@@ -486,13 +466,17 @@ def contact_point_from_vcard(obj: RdfResource) -> tuple[str | None, str | None,
 
 def contact_point_from_foaf(obj: RdfResource) -> tuple[str | None, str | None, str | None]:
     contact_point_org = obj.value(ORG.memberOf)
+    contact_point_member = obj.value(FOAF.member)
```
Comment (streino) [inline udata/rdf.py:469]: This corner case was fixed (and tests updated) when switching to type-based. Reverting to role-based broke the fix.

**Comment by streino** [comment]
Discussed with @ThibaudDauce, we're going for this alternative.
